runtime/runtime_test.go

```diff
@@ -0,0 +1,26 @@
+package config
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestAssetDir(t *testing.T) {
+	t.Parallel()
+	// Change working directory to runtime if needed
+	cwd, err := os.Getwd()
+	assert.NoError(t, err)
+	if !strings.Contains(cwd, "runtime") {
```

Comment:
Is changing directories at all necessary? Shouldn't the syntax files be embedded?

Reply:
Hm, good point. My intention was to provide some sort of a fallback if rexursive tests fail to enter the directory (shouldn't happen though, will check for myself and update this PR if necessary).

Reply:
I have removed the checks